Tests/mock_server.py

```diff
@@ -215,9 +222,12 @@ def configure_proxy_in_demisto(client, proxy=''):
             'data': system_conf,
             'version': -1
         }
-        response = demisto_client.generic_request_func(self=client, path='/system/config',
-                                                       method='POST', body=data)
-        # client.api_client.pool.close()
+        try:
```

Comment:
no need for the try-except, in case this fails we want to stop.
otherwise, the rest of the tests will run without proxy